feature/applist/src/main/java/com/merxury/blocker/feature/applist/AppListScreen.kt

```diff
@@ -75,6 +75,7 @@ fun AppListRoute(
     val errorState by viewModel.errorState.collectAsStateWithLifecycle()
     val warningState by viewModel.warningState.collectAsStateWithLifecycle()
     val appList = viewModel.appListFlow.collectAsState()
+    var isOpenBottomSheet by rememberSaveable { mutableStateOf(false) }
```

Comment:
```suggestion
    var isBottomSheetOpened by rememberSaveable { mutableStateOf(false) }
```

Reply:
BTW I think you put this parameter in a wrong place. You need to put this variable `isBottomSheetOpened` to where the `ModalBottomSheet` will appear, so you don't need to create those two variables to judge the state of the bottomsheet.

Reply:
It's for testing. If put into ModalBottomSheet, cannot set the state to test.